ingredient/ingredient-postgresql/README.md

```diff
@@ -0,0 +1,94 @@
+## Changelogs
+
+* [@azbake/ingredient-postgresql]
+
+## Overview
+
+The PostgreSQL ingredient is a plugin for bake. When included in a recipe, this will create an [Azure Database for PostgreSQL server](https://docs.microsoft.com/en-us/azure/postgresql/).
+This only creates the core account. Individual applications are responsible for creating and maintaining Containers / Databases / Collections.
```

Comment:
By "core account" I'm guessing you mean the server itself or is there more to it than that?

Reply:
yeah, I copied this from the cosmosdb readme which used that terminology. I'll clear that up, your guess is correct